// tbclient.GetForumDetail.GetForumDetailResIdl
syntax = "proto3";

import "Error.proto";

message GetForumDetailResIdl {
    Error error = 1;
    message DataRes {
        message RecommendForumInfo {
            string avatar = 1;
            uint64 forum_id = 2;
            string forum_name = 3;
            uint32 member_count = 5;
            uint32 thread_count = 6;
            string slogan = 7;
            string avatar_origin = 20;
        }
        RecommendForumInfo forum_info = 1;
        message ManagerElectionTab {
            string new_strategy_text = 5;
        }
        ManagerElectionTab election_tab = 8;
    }
    DataRes data = 2;
}
